pkg/transforms/common.go

```diff
@@ -547,27 +547,45 @@ func edgesToApplication(nodeInfo NodeInfo, ns NodeStore, UID string, onlyApplica
 	return ret
 }
 
+// relatedObject stores identifying information for a kubernetes resource.
+// When marshalled to json, the struct names are small to reduce total size.
 type relatedObject struct {
-	kind      string
-	namespace string
-	name      string
+	Group     string `json:"g,omitempty"`
+	Version   string `json:"v,omitempty"`
+	Kind      string `json:"k,omitempty"`
+	Namespace string `json:"ns,omitempty"`
+	Name      string `json:"n,omitempty"`
+}
```

Comment:
@jlpadilla what do you think of this? The shorter field names save up to 23 characters per object. Maybe it's not worth it.

Reply:
It's harder to read, but I like that it makes it harder to incorrectly match the string in a keyword search.

Reply:
Oh, that's a really good point! Accidental feature :smile:

Reply:
I'm thinking that a base64 encoded string could be even better, but harder to decode. Thoughts?

Reply:
base64 has an encoding overhead. Since this is normal text, I'm not sure what we would gain?
```
❯ echo '{"g":"operators.coreos.com","v":"v1alpha1","k":"ClusterServiceVersion","ns":"open-cluster-management","n":"advanced-cluster-management.v2.9.0"}' | wc -c
144

❯ echo '{"g":"operators.coreos.com","v":"v1alpha1","k":"ClusterServiceVersion","ns":"open-cluster-management","n":"advanced-cluster-management.v2.9.0"}' | base64 | wc -c
195
```

Reply:
Well, my thought was that base64 would make it harder to get a match by keyword searches, but maybe it isn't worth the overhead. Another con is that it would be harder to explain when we do get an unexpected match.
Let's keep it as is.